libpromises/dbm_api.c

```diff
@@ -66,6 +76,7 @@ struct DBCursor_
 static pthread_mutex_t db_handles_lock = PTHREAD_ERRORCHECK_MUTEX_INITIALIZER_NP; /* GLOBAL_T */
 
 static DBHandle db_handles[dbid_max] = { { 0 } }; /* GLOBAL_X */
+static DynamicDBHandles *db_dynamic_handles;
```

Comment:
This is needed as we need one DB per package module. We can not know how many of those will be used in policy. In fact dynamic is static from the durability perspective. It is dynamic from the number of instances perspective.